sale_automatic_workflow/tests/common.py

```diff
@@ -80,6 +80,7 @@ def create_full_automatic(self, override=None):
                 "create_invoice": True,
                 "validate_invoice": True,
                 "invoice_date_is_order_date": True,
+                "register_payment": True,
```

Comment:
@hoangtrann IMHO this should go in an override of common in the 'job' module. This will fixes tests too I think